app/src/testPsi/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCaptureRepositoryFunctionsTest.kt

```diff
@@ -1,4 +1,4 @@
-package org.dhis2.usecases.eventsWithoutRegistration.eventCapture
+package org.dhis2.usescases.eventsWithoutRegistration.eventCapture
```

Comment:
why in plural "usescases"? I would have said that the plural of usecase is usecases...isn't it?

Reply:
It's to follow the same approach that main directory:
package org.dhis2.usescases